### PR TITLE
Update Rpc.proto

### PR DESCRIPTION
**Pull request by JayArrowz**
Modify BroadcastRawTransaction messages

### CODE REVIEW

File: src/Rpc.proto [hunk 1]
```diff
@@ -123,11 +124,11 @@ message DecodeRawTransactionResponse {
 }
 
 message BroadcastRawTransactionRequest {
-    bool query = 1;
+	Transaction.TransactionBroadcast transaction = 1;
 }
 
 message BroadcastRawTransactionResponse {
-    string query = 1;
+    bytes responseCode = 1;
```
Comment (monsieurleberre) [inline src/Rpc.proto]: Enum ? 🙄😬😬

**Review by monsieurleberre — DISMISSED** [review]
OK anyway

**Review by monsieurleberre — APPROVED** [review]
OK cool :)
So you want to do the Add / Get file response in another one?

**Review by Millymanz — APPROVED** [review]
Nice idea

**Pull request merged**